src/components/home/Button.jsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
why is this rule disabled everywhere ?

Reply:
@ericus123 The way I was destructuring props works but Eslint was giving errors.